frontend/pages/index.page.tsx

```diff
@@ -292,37 +255,27 @@ const SubscriptionBox: FC = () => {
   );
 
   return (
-    <Box
-      sx={{
-        display: "flex",
-        alignItems: "center",
-        justifyContent: "center",
-        margin: "1.5rem 1rem 1.5rem 1rem",
-      }}
-      hidden
-    >
-      <Card elevation={5}>
-        <CardContent>
-          <Grid
-            item
-            sx={{
-              alignItems: "center",
-              display: "flex",
-              justifyContent: "space-between",
-            }}
-          >
-            <Typography style={{ fontSize: "20pt", fontWeight: "bold" }}>
-              Sign up for the weekly ModularHistory newsletter
-            </Typography>
-            <Button>
-              <CloseIcon />
-            </Button>
-          </Grid>
-          <Typography>Sign up to recieve our newsletter!</Typography>
-          {subscriptionForm}
-          {subscribeDisclaimer}
-        </CardContent>
-      </Card>
-    </Box>
+    <StyledCard elevation={5}>
```

Comment:
This isn't a problem necessarily, but we should consider using the `raised` prop instead of `elevation={magicNumber}` if it looks similar enough.